tests/apply.test.js

```diff
@@ -1328,3 +1328,54 @@ it('should be possible to use apply in plugins', async () => {
     `)
   })
 })
+
+it('The apply class cache is invalidated when rules change', async () => {
```

Comment:
Nitpick: can you make this read like a sentence `it('should ...')` or `test('the apply class...')`